aws/aws_test.go

```diff
@@ -6,9 +6,11 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var validPolicy = api.NewPolicy("name", "testns", []api.StatementSpec{
-	{Resource: "arn:aws:s3:::my_corporate_bucket/exampleobject.png", Action: []string{"an:action"}},
-})
+var (
+	validPolicy = api.NewPolicy("name", "testns", []api.StatementSpec{
+		{Resource: "arn:aws:s3:::my_corporate_bucket/exampleobject.png", Action: []string{"an:action"}},
+	})
+)
```

Comment:
Out of curiosity, what does this change?

Reply:
Not much, during dev I added another variable so I grouped them in a var block, and them removed it, so the block is still there. I could revert this change, but IMO it's fine to keep it like this as well